shop.py

```diff
@@ -128,6 +128,8 @@ def update_shopify(self):
                 ])
             if existent_shipment:
                 continue
+            if not hasattr(order, 'shipping_address'):
+                order.shipping_address = None
```

Comment:
It is strange to add a field to the object. I think we'd rather:

```
# In some cases (ie, the customer will pick the product from the warehouse) the shipping_address may not exist
shipping_address = getattr(order, 'shipping_address', None)
```

This will require updating the code in get_customer_phone_numbers to use getattr(...).